doc/sailfish-webview-qmozcontext.cpp

```diff
@@ -0,0 +1,367 @@
+    registered an observer for \a topic.
+
+    Additional data can be sent with the notification using the \a value
+    parameter encoded as a JSON string.
+
+    \sa addObserver, recvObserve
+*/
+void QMozContext::notifyObservers(const QString &topic, const QString &value);
+
+/*!
+    \brief Sends a broadcast notification that can be observed by other components.
+
+    Sends a notification that will be picked up by any component that has
+    registered an observer for \a topic.
+
+    Additional data can be sent with the notification using the \a value
+    parameter which should contain a structure (e.g. a \c{QVariantMap}) that will
+    be encoded as a JSON object.
+
+    \sa addObserver, recvObserve
+*/
+void QMozContext::notifyObservers(const QString &topic, const QVariant &value);
+
+/*!
+    \brief Starts the engine event loop.
+
+    When WebEngine is instantiated as a QML component this is called
+    automatically during initalisation. However, it can be called earlier to
+    set the WebEngine event view running in case the intialization process is
+    being overridden.
+
+    See \l {SailfishOS::WebEngine::initialize}{WebEngine::initialize} for more info.
+
+    The \a aDelay allows for an asynchronous startup. Calling without the
+    \a aDelay parameter specified will cause execution to happen on the nested
+    main loop, which will block until \l stopEmbedding is called.
+
+    \sa {SailfishOS::WebEngine::initialize}{WebEngine::initialize}, stopEmbedding, setProfile
+*/
+void QMozContext::runEmbedding(int aDelay = -1);
+
+/*!
+    \brief Stops the engine event loop.
+
+    Causes execution of the WebEngine to halt. If \l runEmbedding was called
+    without an \c aDelay parameter, this will cause the \l runEmbedding call to
+    unblock.
+
+    \sa {SailfishOS::WebEngine::initialize}{WebEngine::initialize}, runEmbedding, setProfile
+*/
+void QMozContext::stopEmbedding();
+
+/*!
+    \internal
+    \brief Sends the "final-ui-startup" notification to observers
+
+    For internal use? I don't actually see it used anywhere.
+*/
+void QMozContext::notifyFirstUIInitialized();
+
+/*!
+    \brief Sets the location for the profile directory.
+
+    This is called during the startup process to set \a profilePath as the
+    location to store the profile (inside which the \c{.mozilla} directory will
+    be created). By default this is set to
+    \c QStandardPaths::writableLocation(QStandardPaths::CacheLocation), which
+    is usually \c{~/.cache/<organisation name>/<application name>}.
+
+    Generally there is no need to call \c setProfile as the profile will
+    automatically set up on initialization, and can also be passed as
+    parameter to \l{WebEngine::initialize}.
+
+    When WebEngine is instantiated as a QML component this is also called
+    automatically during initalisation. However, it can be called earlier to
+    set a custom profile location in case the intialization process is being
```

Comment:
it's a difficult one to spell ;-)

Reply:
Thanks for highlighting them all. Not sure why my fingers have so much trouble with it.